plugins/backend/catalog-backend-module-aws/src/providers/AWSS3BucketProvider.ts

```diff
@@ -94,8 +94,15 @@ export class AWSS3BucketProvider extends AWSEntityProvider {
       if (bucket.Name) {
         const bucketArn = `arn:aws:s3:::${bucket.Name}`;
         const consoleLink = new ARN(bucketArn).consoleLink;
-        const tagsResponse = await s3.getBucketTagging({ Bucket: bucket.Name });
-        const tags = tagsResponse?.TagSet ?? [];
+        let tags: Tag[] = [];
+        try {
+          const tagsResponse = await s3.getBucketTagging({
+            Bucket: bucket.Name,
+          });
+          tags = tagsResponse?.TagSet ?? [];
+        } catch (e) {
+          this.logger.debug('No tags found for bucket');
```

Comment:
hmm. I guess you dont know if this is the cause of the exception. I would use: failed to retrieve any tags for the bucket. Also no harm in putting in a few metadata items, like the bucket name maybe.

Reply:
Thanks, updated